frontend/src/App.vue

```diff
@@ -48,4 +48,8 @@ export default {
   -moz-osx-font-smoothing: grayscale;
   color: #2c3e50;
 }
+
+.page-container {
+  min-height: calc(100vh - 65px - 128px);
```

Comment:
How is that calculated, CSS Ninja :D?

Reply:
I take current height if header and current height of footer and subtract them from height of the screen. Currently those heights don't change on desktop, but I am waiting for the app to get connected to the api to see how many elements we will have on nav.
Once we are designing mobile we will likely update this too.